Description: Loosen thiserror version requirement
Author: Abhinaba Rakshit <abhinaba.rakshit@oss.qualcomm.com>
Origin: other, Debian packaging
Last-Update: 2026-04-22
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: mink-idl-compiler/idlc_ast/Cargo.toml
===================================================================
--- mink-idl-compiler.orig/idlc_ast/Cargo.toml
+++ mink-idl-compiler/idlc_ast/Cargo.toml
@@ -12,7 +12,7 @@ description = "AST structure for IDL; co
 idlc_errors = {path = "../idlc_errors" }
 pest = "2.7.9"
 pest_derive = "2.7.9"
-thiserror = "1.0.59"
+thiserror = "1.0.50"
 
 [package.metadata.workspaces]
 independent = true
Index: mink-idl-compiler/idlc_ast_passes/Cargo.toml
===================================================================
--- mink-idl-compiler.orig/idlc_ast_passes/Cargo.toml
+++ mink-idl-compiler/idlc_ast_passes/Cargo.toml
@@ -9,7 +9,7 @@ description = "Contains compiler passes
 [dependencies]
 "idlc_ast" = { path="../idlc_ast" }
 idlc_errors = {path = "../idlc_errors" }
-thiserror = "1.0.59"
+thiserror = "1.0.50"
 
 [dev-dependencies]
 serial_test = "*"
Index: mink-idl-compiler/idlc_mir_passes/Cargo.toml
===================================================================
--- mink-idl-compiler.orig/idlc_mir_passes/Cargo.toml
+++ mink-idl-compiler/idlc_mir_passes/Cargo.toml
@@ -10,7 +10,7 @@ idlc_ast = { path = "../idlc_ast" }
 idlc_ast_passes = { path="../idlc_ast_passes" }
 idlc_errors = {path = "../idlc_errors" }
 idlc_mir = { path="../idlc_mir" }
-thiserror = "1.0.59"
+thiserror = "1.0.50"
 
 [package.metadata.workspaces]
 independent = true
